packages/graphback-runtime-knex/package.json

```diff
@@ -39,10 +39,10 @@
   "dependencies": {
     "@graphback/core": "0.15.1",
     "dataloader": "2.0.0",
-    "graphql-subscriptions": "1.1.0",
-    "knex": "0.21.4"
+    "graphql-subscriptions": "1.1.0"
   },
   "peerDependencies": {
+    "knex": "0.20.x || 0.21.x",
```

Comment:
Will this work if `0.22.0` is released?

Reply:
It will give a warning, but it will still work yes. We could add 0.22.x || 0.23.x

Reply:
Thanks